web-admin/src/routes/[organization]/[project]/+layout.svelte

```diff
@@ -1,31 +1,142 @@
+<script context="module" lang="ts">
+  const PollTimeWhenProjectDeploymentPending = 1000;
+  const PollTimeWhenProjectDeploymentError = 5000;
+  const PollTimeWhenProjectDeploymentOk = RUNTIME_ACCESS_TOKEN_DEFAULT_TTL / 2; // Proactively refetch the JWT before it expires
+
+  const baseGetProjectQueryOptions: QueryObserverOptions<
```

Comment:
Any reason to no move this to a .ts file?

Reply:
I don't feel too strongly. It only relates to the Query Observers in this `+layout.svelte` file, so this felt like a middle-ground between code co-location & factoring it out.